### PR TITLE
[IWT17] Tutorial page

### PR DESCRIPTION
**Pull request by Eleara**
This PR adds the tutorial page.

### CODE REVIEW

File: app/javascript/pages/help/components/help.jsx [hunk 1]
```diff
@@ -1,7 +1,16 @@
 import React from "react";
+import AboutSection from "./about-section";
+import GettingStartedSection from "./getting-started-section";
+import GroundSection from "./ground-section";
 
 const Help = () => {
-  return <div>Tutorial page.</div>;
+  return (
+    <div className='form-wrapper'>
```
Comment (karczewskiPiotr) [inline app/javascript/pages/help/components/help.jsx:8]: This is not a form, name it differently. 💁‍♀️

File: app/javascript/pages/help/components/about-section.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const AboutSection = () => {
+  return (
+    <div className='form'>
```
Comment (karczewskiPiotr) [inline app/javascript/pages/help/components/about-section.jsx:5]: Change to a different name.

File: app/javascript/pages/help/styles/form.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    .title {
+      font-weight: bold;
+      font-size: 24px;
+      margin-bottom: 10px;
+    }
+    p {
+      margin-bottom: 15px;
+    }
+    .image {
+      width: 100%;
```
Comment (karczewskiPiotr) [inline app/javascript/pages/help/styles/form.scss:22]: Without automatic height? 🤔

Comment (Eleara) [reply app/javascript/pages/help/styles/form.scss:22]: It scales on its own.

Comment (karczewskiPiotr) [reply app/javascript/pages/help/styles/form.scss:22]: magic! 🧙

File: app/javascript/pages/help/index.scss [hunk 4]
```diff
@@ -1,5 +1,6 @@
 @import "../../helpers/variables.scss";
 @import "../../helpers/alerts.scss";
+@import "./styles/form.scss";
```
Comment (karczewskiPiotr) [inline app/javascript/pages/help/index.scss:3]: !

**Pull request merged**